### ISSUE TITLE
How can I use the pre-trained model for testing

### ISSUE THREAD

**Issue by ferric123** [scaelles/OSVOS-TensorFlow#25]
Hi, I am interested in this work and want to take look at the test result first. Could you please give me an instruction on how I can make use of your fine-tuned pre-trained model(if there is one could you put the link) to feed an arbitrary test image and see the segmented output?

**Comment by scaelles** [comment]
Hello,
I would recommend that you start in the [osvos_demo.py](https://github.com/scaelles/OSVOS-TensorFlow/blob/master/osvos_demo.py) script. You have to input to the algorithm at least one frame with the corresponding annotated binary mask. After training for a while, you will be able to test with the rest of the frames of the video sequence.
Let us know if you have any further question!

**Comment by ferric123** [comment]
> Hello,
> I would recommend that you start in the [osvos_demo.py](https://github.com/scaelles/OSVOS-TensorFlow/blob/master/osvos_demo.py) script. You have to input to the algorithm at least one frame with the corresponding annotated binary mask. After training for a while, you will be able to test with the rest of the frames of the video sequence.
> Let us know if you have any further question!

Okay,I will try that, thanks in advance!
